test/test-queue-that.js

```diff
@@ -310,6 +310,65 @@ describe('createQueueThat', function () {
       expect(options.process.callCount).to.be(3)
     })
 
+    it('should report repeated items on process error', function () {
+      localStorageAdapter.setQueue(_.range(4))
```

Comment:
Oh, interesting. So we're considering a error callback a potential cause for "duplicates"?